code-of-conduct-enforcement/cpg-onboarding.md

```diff
@@ -12,7 +12,7 @@ The behaviors that informed this ban with clear steps to avoid in future are des
 
 * Violation 1  - [CPG exact wording] , ways to avoid
 * Violation 2  - [CPG exact wording] , ways to avoid
-* Violation 3  - [CPG exact wording] , aways to avoid
```

Comment:
Finding ways to avoid their duties which causes them to get violations and more than one violations and be cause for more serious consequences later.